traffic_portal/app/src/common/api/CoordinateService.js

```diff
@@ -17,66 +17,60 @@
  * under the License.
  */
 
-var CoordinateService = function($http, $q, Restangular, locationUtils, messageModel, ENV) {
+var CoordinateService = function($http, locationUtils, messageModel, ENV) {
 
     this.getCoordinates = function(queryParams) {
```

Comment:
not sure what's up here but https://localhost:8442/#!/coordinates looks like

![image](https://user-images.githubusercontent.com/251272/58210519-d74bdf80-7ca6-11e9-95ca-3d0efbb1b563.png)

Reply:
I think I fixed that - will check in a second

Reply:
yep, fixed.